asteroid/filterbanks/transforms.py

```diff
@@ -342,6 +345,17 @@ def from_mag_and_phase(mag, phase, dim: int = -2):
     return torch.cat([mag * torch.cos(phase), mag * torch.sin(phase)], dim=dim)
 
 
+# Alias
+from_polar = from_mag_and_phase
```

Comment:
We should also add these aliases in `complex_nn` IMO or stick to any of the two names everywhere.

Reply:
Hm ok this is harder than I thought 😬 so want to have 4 things:

- “good” (meaningful) names
- consistent with torchaudio 
- consistent within Asteroid
- compatible with current Asteroid names

We’d probably have to put a non trivial amount of effort into getting all of these right...

I don’t know what to do really.

Reply:
See #335